storage/tables/data-types/index.md

```diff
@@ -992,3 +992,19 @@ The `TIMESTAMP` base type represents a date value with a time portion.
     <th>Target</th>
 </tr>
 </table>
+
+### Native datatypes
+
+As mentioned above, Keboola stores data in storage as text by default. Native datatypes feature breaks this paradigm. If this feature is enabled, data in storage are stored in columns which have datatypes as described in Keboola metadata. 
+

```

Comment:
Tady bych to neokecával tolik, je to dokumentace ne Danielle Steel

```
If this feature is enabled, table columns in storage can have datatypes supported by backend.
```

Reply:
nevim, par radku, ktere to uvadeji do kontextu mi asi zily netrhaji. Zvlast, kdyz nad tim je wall of tables s basetypama